### DISCUSSION TITLE
What's your favorite sddm theme

### DISCUSSION THREAD

**Discussion by mylinuxforwork** [mylinuxforwork/dotfiles#312] [General]
HI all,

I want to replace the sugar candy theme for the next version because it's not maintained anymore. What are your favorite themes for sddm?

**Comment by ap27** [comment]
[(https://github.com/minMelody/sddm-sequoia)]
with background
[(https://github.com/mylinuxforwork/wallpaper/blob/main/wallhaven3.jpg)]
